lib/cocoapods-binary-cache/pod-rome/xcodebuild_command.rb

```diff
@@ -61,7 +62,8 @@ def make_up_build_args(args)
       args_[:simulator] ||= []
       args_[:device] ||= []
       args_[:default].prepend("BITCODE_GENERATION_MODE=bitcode") if bitcode_enabled?
-      args_[:default].prepend("DEBUG_INFORMATION_FORMAT=dwarf") if disable_dsym?
+      debug_format = disable_dsym? ? "dwarf" : "dwarf-with-dsym"
```

Comment:
Actually, it was purposely unset if `disable_dsym?` is false.
The flag `disable_dsym` is more like `force disable dsym`.
I think later, we should remove this option and the xcodebuild should respect the build settings in the project.

I don't have any objection to this change though.

Reply:
Sounds right, I reverted this logic 👍